### DISCUSSION TITLE
apisix的jwt-auth动态token，配置后，单个请求验证没问题，当请求多时就会出现200和401的问题。同一个路由。为什么，各位大佬求解

### DISCUSSION THREAD

**Discussion by IT-Forerunner** [apache/apisix#11062] [Q&A]
apisix的jwt-auth动态token，在请求多时会间接出现401的问题，是用一个路由，且tk是正确的。出现401的返回是{"message":"failed to verify jwt"}，但是tk确实没问题。用一个路由，有的200 有的401 。验证过不是用tk,请求没问题。也验证过key-auth（静态tk）也没问题，就是jwt-auth会出现这个问题。各位大佬求解
![QQ图片20240319174639](https://github.com/apache/apisix/assets/116343416/b3710aa5-34f9-44ca-9e78-008efc6f9c83)

**Comment by yy217** [comment]
try add param secret
